tox.ini

```diff
@@ -3,17 +3,13 @@ envlist = py{27,35}, py{27,35}-marionette, flake8
 
 [testenv]
 passenv = DISPLAY PYTEST_ADDOPTS
-deps =
-    mock
-    pytest==3.0.4
-    pytest-selenium
-    pytest-xdist==1.15.0
+deps = -rrequirements/tests.txt
```

Comment:
@frankrousseau no worries; I appreciate you taking a look at the pull request - thanks!